testflinger_agent/client.py

```diff
@@ -95,8 +96,7 @@ def repost_job(self, job_data):
         )
         self.post_live_output(job_id, job_output)
         try:
-            session = self._requests_retry(retries=5)
```

Comment:
moved to class attribute to be used in new method (post_agent_data)
see below for reasoning